### ISSUE TITLE
[Editor] toolbar button's state not updated after click on them (only after click somewhere else)

### ISSUE THREAD

**Issue by sevaru** [telerik/kendo-angular#3975]
**Describe the bug**
Click on toolbar button add formatting to text but do not show it's selected state

**To Reproduce**
1) Select text
2) Click on (bald, utalic or any other formatting tool)

https://stackblitz.com/edit/angular-1qpfbn?file=src/app/app.component.ts

**Expected behavior**
Toolbar button is "selected"

**Actual behavior**
Toolbar button is not selected until any click inside toolbar or editor

**Screenshots**
https://user-images.githubusercontent.com/12616521/231783510-6bd786f4-0dba-46e1-a3e6-15cf890d3c28.mp4

**Browser**
 - OS: Windows 10
 - Browser Chrome
 - Version 112.0.5615.87

**Additional context**
Angular: 15.2.6 (onPush)
@progress/kendo-angular-editor: 11.6.0

**Comment by yanmariomenev** [comment]
Duplicate of - https://github.com/telerik/kendo-angular/issues/3947
